includes/Core/Authentication/Helpers/Block_External.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Authentication\Helpers\Block_External
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2025 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Authentication\Helpers;
+
+use Google\Site_Kit\Core\Util\URL;
+use Google\Site_Kit_Dependencies\GuzzleHttp\Exception\RequestException;
+use WP_Http;
+
+/**
+ * Class for blocking external requests.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Block_External {
+
+	/**
+	 * Guzzle Handler for blocking external requests.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return callable A Guzzle handler that blocks external requests.
```

Comment:
We can model the way they're documented in Guzzle. Method short descriptions are always supposed to start with a specific verb tense (e.g. "Gets") which I don't think is strictly enforced in PHP but something to follow for consistency 👍